recipes/qcoro/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class QCoroConan(ConanFile):
+    name = "qcoro"
+    license = "MIT"
+    homepage = "https://github.com/danvratil/qcoro"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "C++ Coroutines for Qt."
+    topics = ("coroutines", "qt")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "asan": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "asan": False,
+    }
+    generators = "cmake", "cmake_find_package_multi"
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        minimum_versions = {
+                "gcc": "10",
+                "Visual Studio": "17",
+                "msvc": "19.29",
+                "clang": "8",
+                "apple-clang": "13"
+        }
+        return minimum_versions
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def build_requirements(self):
+        self.build_requires("cmake/3.21.3")
+
+    def requirements(self):
+        self.requires("qt/6.2.2")
+        self.requires("expat/2.4.2", override=True)
```

Comment:
Why expat version is overridden?

Reply:
I don't remember. probably a conflict in qt requirements ?

Reply:
Would be nice to remove it in the next PR.